kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -118,6 +118,10 @@ class KyuubiSessionImpl(
     // we should call super.open before running launch engine operation
     super.open()
 
+    operationalLogRootDir.foreach { dir =>
+      Option(sessionManager.tempFileService).foreach(_.addPathToExpiration(dir))
```

Comment:
`sessionManager.tempFileService` should always be present, right?

Reply:
Yes. You're right. `sessionManager.tempFileService` is always present, let me skip the option wrapper here.